apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2Test.java

```diff
@@ -188,6 +189,38 @@ public void mapEobCommonGroupInpOutHHAHospiceSNFWhenNoFiNumberExpectNoFiNumExten
     assertNull(fiNumExtension);
   }
 
+  /**
+   * Verifies that {@link
+   * gov.cms.bfd.server.war.stu3.providers.TransformerUtils#careTeamMatchingExtensions(
+   * (org.hl7.fhir.dstu3.model.ExplanationOfBenefit.CareTeamComponent, String, String)} verifies if
+   * an extension is found
+   */
+  @Test
+  public void careTeamMatchingExtensionsTest() {
+    String referenceUrl = "http://test.url";
+    String codeValue = "code";
+    Coding coding = new Coding();
+    coding.setCode(codeValue);
+    Extension extension = new Extension(referenceUrl);
+    extension.setValue(coding);
+    CareTeamComponent careTeamComponent = new CareTeamComponent();
+    careTeamComponent.addExtension(extension);
+
+    boolean returnResult =
+        TransformerUtilsV2.careTeamMatchingExtensions(careTeamComponent, referenceUrl, codeValue);
+
+    assertTrue(returnResult);
+
+    returnResult = TransformerUtilsV2.careTeamMatchingExtensions(careTeamComponent, "", codeValue);
+
+    assertFalse(returnResult);
+
+    returnResult =
+        TransformerUtilsV2.careTeamMatchingExtensions(careTeamComponent, "http://test.url", "");
+
+    assertFalse(returnResult);
+  }
```

Comment:
These should all really be separate tests, for clarity if one fails.

Reply:
separated into separate tests